src/Nethermind/Chains/chiado.json

```diff
@@ -73,6 +73,7 @@
     "eip4844TransitionTimestamp": "0x65ba8e4c",
     "eip5656TransitionTimestamp": "0x65ba8e4c",
     "eip6780TransitionTimestamp": "0x65ba8e4c",
+    "gipxyzTransitionTimestamp": "0x65efbeef",
```

Comment:
We are not enabling anything yet, so I would say remove it for now, unless this is not mergeable until time is decided